modules/programs/mako.nix

```diff
@@ -0,0 +1,312 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.programs.mako;
+
+in {
+  options = {
+    programs.mako = {
+      enable = mkEnableOption ''
+        Mako, lightweight notification daemon for Wayland
+      '';
+
+      maxVisible = mkOption {
+        default = 5;
+        type = types.nullOr types.int;
+        description = ''
+          Set maximum number of visible notifications. Set -1 to show all.
+        '';
+      };
+
+      sort = mkOption {
+        default = "-time";
+        type =
+          types.nullOr (types.enum [ "+time" "-time" "+priority" "-priority" ]);
+        description = ''
+          	        Sorts incoming notifications by time and/or priority in ascending(+)
+                    or descending(-) order.
+          	      '';
+      };
+
+      output = mkOption {
+        default = null;
+        type = types.nullOr types.str;
+        description = ''
+          	        Show notifications on the specified output. If empty, notifications
+                    will appear on the focused output. Requires the compositor to support
+                    the Wayland protocol xdg-output-unstable-v1 version 2.
+                  '';
+      };
+
+      layer = mkOption {
+        default = "top";
+        type =
+          types.nullOr (types.enum [ "background" "bottom" "top" "overlay" ]);
+        description = ''
+          	        Arrange mako at the specified layer, relative to normal windows.
+                    Supported values are background, bottom, top, and overlay. Using
+                    overlay will cause notifications to be displayed above fullscreen
+                    windows, though this may also occur at top depending on your
+                    compositor.
+                  '';
+      };
+
+      anchor = mkOption {
+        default = "top-right";
+        type = types.nullOr (types.enum [
+          "top-right"
+          "top-center"
+          "top-left"
+          "bottom-right"
+          "bottom-center"
+          "bottom-left"
+          "center"
+        ]);
+        description = ''
+          Show notifications at the specified position on the output.
+          Supported values are top-right, top-center, top-left, bottom-right,
+          bottom-center, bottom-left, and center.
+        '';
```

Comment:
...to this.

End quotes aligned with `description`, body aligned with `description` + 2 spaces.